scss/mixins/_arrow.scss

```diff
@@ -17,37 +17,37 @@
 // @private mixin arrow-base
 // Used to set up an arrow element
 @mixin __arrow-base($size: 10px) {
-  display: inline-block;
-  width: 0;
-  height: 0;
+    display: inline-block;
+    width: 0;
+    height: 0;
 
-  $size: $size/2;
-  border: $size solid transparent;
+    $size: $size/2;
```

Comment:
`$size/2` should be written with a single space on each side of the operator: `$size / 2`